.github/workflows/test-deploy.yml

```diff
@@ -118,10 +118,9 @@ jobs:
           cp build/cml-macos{-x64,}
       - if: inputs.release
         run:
-          find build -type f | xargs gh release upload $(basename ${{
-          github.head_ref }})
+          find build -type f | xargs gh release upload "$GITHUB_REF_NAME"
         env:
-          GITHUB_TOKEN: ${{ secrets.TEST_GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
~~Is it `GH_TOKEN`? ~~ https://cli.github.com/manual/gh_help_environment

Reply:
> _[GH_TOKEN, GITHUB_TOKEN (in order of precedence)](https://cli.github.com/manual/gh_help_environment)_

Who cares? 🤷🏼‍♂️